bin/MadGraph5_aMCatNLO/cards/WW_leptonic/WWjj_SS_dim8_ewk_qcd_proc_card.dat

```diff
@@ -0,0 +1,38 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.1                 2014-09-25         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model /afs/cern.ch/work/s/shundal/genproductions/bin/MadGraph5_aMCatNLO/models/SM_Ltotal_Ind5v2020v2_UFO
```

Comment:
If the model is not in [1], let me know the path to public repo in lxplus so that I can put it to [1].

[1] https://cms-project-generators.web.cern.ch/cms-project-generators/

Reply:
And after I do,
please add the extramodels card as well

Reply:
And change this line to import model SM_Ltotal_Ind5v2020v2_UFO instead of your private path

Reply:
Model is already there in [1]. Changed import model line.

Reply:
@sihyunjeon @agrohsje Kindly have a look and please let me know in case some more changes are required.